File: src/org/elixir_lang/ElixirScriptFileType.java
package org.elixir_lang;

import org.jetbrains.annotations.NotNull;

import javax.swing.*;

public class ElixirScriptFileType extends ElixirFileType {
    public static final ElixirScriptFileType INSTANCE = new ElixirScriptFileType();

    @NotNull
    @Override
    public String getName() {
        return "Elixir Script";
    }

    @NotNull
    @Override
    public String getDescription() {
        return "Elixir Script file";
    }

    @NotNull
    @Override
    public String getDefaultExtension() {
        return "exs";
    }
}

File: src/org/elixir_lang/ElixirScriptFileType.kt
package org.elixir_lang

class ElixirScriptFileType : ElixirFileType() {
    override fun getName(): String = "Elixir Script"
    override fun getDisplayName(): String = name
    override fun getDescription(): String = "Elixir Script file"
    override fun getDefaultExtension(): String = "exs"

    companion object {
        @JvmField
        val INSTANCE = ElixirScriptFileType()
    }
}
